amundsen_application/static/js/components/HomePage/styles.scss

```diff
@@ -0,0 +1,13 @@
+@import 'variables';
+
+.home-page {
+  .search-list-container {
```

Comment:
This style should have it's own name. In addition we have to increase the spacing between each of the lists on this page as requested by Design.